plugins/react-native.js

```diff
@@ -73,9 +87,12 @@ function reactNativePlugin(Raven) {
 
     var defaultHandler = ErrorUtils.getGlobalHandler && ErrorUtils.getGlobalHandler() || ErrorUtils._globalHandler;
 
+    var rethrow = pluginOptions.rethrow || __DEV__;
```

Comment:
If `rethrow` is false, this will still throw in `__DEV__`.

How about:

```javascript
var rethrow = 'rethrow' in pluginOptions ? pluginOptions.rethrow || __DEV__;
```